tests/conftest.py

```diff
@@ -6,27 +6,6 @@
 from bluesky.run_engine import TransitionError
 
 
-def pytest_addoption(parser):
-    parser.addoption(
-        "--skip-stomp",
-        action="store_true",
-        default=False,
-        help="skip stomp tests (e.g. because a server is unavailable)",
-    )
-
-
-def pytest_configure(config):
-    config.addinivalue_line("markers", "stomp: mark test as requiring stomp broker")
-
-
-def pytest_collection_modifyitems(config, items):
-    if config.getoption("--skip-stomp"):
```

Comment:
Now that we've removed this, we no longer need any special handling for STOMP in CI, can you remove the parts of the github actions that start up activemq and rabbitmq?

Reply:
There are 2 tests that are used to connect to the activemq/rabbitmq...I could delete them or skip both I don't think are suitable ..